src/ksym.rs

```diff
@@ -91,29 +91,69 @@ impl KSymResolver {
         }
     }
 
-    pub fn find_address_ksym(&self, addr: u64) -> Option<&Ksym> {
+    pub fn find_address_ksym(&self, addr: u64) -> Vec<&Ksym> {
```

Comment:
Why are we using plural in one of the names (i.e., renaming it) but not here? Seems inconsistent, given that both methods can now return multiple addresses?

Reply:
agree!